ocp_addons_operators_cli/utils/addons_utils.py

```diff
@@ -104,7 +107,15 @@ def assert_addons_user_input(addons, brew_token):
         assert_missing_managed_odh_brew_token(addons=addons, brew_token=brew_token)
 
 
-def prepare_addons(addons, ocm_token, endpoint, brew_token, install):
+def store_kubeconfig_from_cluster_name(cluster_name, kubeconfig_dict):
+    kubeconfig_path = os.path.join("/tmp", f"kubeconfig-{cluster_name}")
+    with open(kubeconfig_path, "w") as fd:
+        fd.write(yaml.dump(kubeconfig_dict))  # noqa: FCN001
```

Comment:
add 'yaml' to fcn_exclude_functions

Reply:
@rnetser it is already added in .flake8 file https://github.com/RedHatQE/ocp-addons-operators-cli/blob/main/.flake8#L15

Reply:
added `write` to the file and it resolved the pre-commit failure.